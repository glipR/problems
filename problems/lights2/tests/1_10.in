651730652401258729