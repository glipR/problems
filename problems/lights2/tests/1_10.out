434487101600839153
